### PR TITLE
HTSlib 1.4.1 for foss 2016a

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
HTSlib 1.4.1 for foss 2016a

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4f5d00073451af73cc7011e950bc4d74 for a full test report.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb]: @FokkeDijkstra Please fix this URL to point to the new `easybuilders` repo.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+version = '1.4.1'
+
+homepage = "http://www.htslib.org/"
+description = """A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb]: Please add a checksum.

Comment (FokkeDijkstra) [reply easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb]: Checksum has been added.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+checksums = ['e549fee04d95c1cf8f46347e9e9bbd93']
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb]: @FokkeDijkstra Do you mind making this a `SHA256` checksum? Recent EasyBuild versions can auto-detect whether the checksum is `MD5` or `SHA256`, cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#checksums

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2628.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a37b54fdc3e299faddbc8b26f2684232 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2038.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/74340bdac475bacfaa7175f7e1d68b16 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @FokkeDijkstra!

**Pull request merged**